scss/reprivata-theme/custom.scss

```diff
@@ -71,17 +71,18 @@ $theme-colors: (
   "dark": $dark,
 );
 
-$scarlet-100: tint-color($scarlet, 80%);
-$scarlet-200: tint-color($scarlet, 60%);
-$scarlet-300: tint-color($scarlet, 40%);
-$scarlet-400: tint-color($scarlet, 20%);
-$scarlet-500: $scarlet;
-$scarlet-600: shade-color($scarlet, 20%);
-$scarlet-700: shade-color($scarlet, 40%);
-$scarlet-800: shade-color($scarlet, 60%);
-$scarlet-900: shade-color($scarlet, 80%);
+// $scarlet-100: tint-color($scarlet, 80%);
+// $scarlet-200: tint-color($scarlet, 60%);
+// $scarlet-300: tint-color($scarlet, 40%);
+// $scarlet-400: tint-color($scarlet, 20%);
+// $scarlet-500: $scarlet;
+// $scarlet-600: shade-color($scarlet, 20%);
+// $scarlet-700: shade-color($scarlet, 40%);
+// $scarlet-800: shade-color($scarlet, 60%);
+// $scarlet-900: shade-color($scarlet, 80%);
 
 // spacer
+$spacer: 16px;
```

Comment:
Curious as to why we're using `px` instead of `rem`

Reply:
Because we were initially using pixel values for spacing (with Styled System), however I can update it so that we're using `rem` instead.